src/darwin/frida-helper-backend-glue.m

```diff
@@ -825,15 +832,13 @@ static gboolean frida_agent_context_init_functions (FridaAgentContext * self, Gu
   thread_act_array_t threads;
   guint thread_index;
   mach_msg_type_number_t thread_count = 0;
-  GumDarwinUnifiedThreadState state;
-  mach_msg_type_number_t state_count = GUM_DARWIN_THREAD_STATE_COUNT;
   thread_state_flavor_t state_flavor = GUM_DARWIN_THREAD_STATE_FLAVOR;
-  GumAddress dyld_start, dyld_granularity, dyld_chunk, dyld_header;
-  GumDarwinModule * dyld;
-  GumAddress dyld_init_address;
+  GumAddress libc_header = 0;
```

Comment:
Let's move the initialization down to right before it's used, to make it a bit clearer. Also feel free to merge this line and the next to avoid repeating the type (since the variables are related).